Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -131,7 +131,7 @@ public BoundsControlActivationType BoundsControlActivation
         /// <summary>
         /// Check to draw a tether point from the handles to the hand when manipulating.
         /// </summary>
-        public bool DrawTetherWhenManipulating => drawTetherWhenManipulating;
+        public bool DrawTetherWhenManipulating { get => drawTetherWhenManipulating; set => drawTetherWhenManipulating = value;}
```

Comment:
nit: it's easier to read these on separate lines

Reply:
i usually prefer this way for default get / set bc it doesn't take up so much space :D

Reply:
but you're right - actually annoys me sometimes as well that i have to scroll to the right :D - fixed